### DISCUSSION TITLE
Le futur d'Olop

### DISCUSSION THREAD

**Discussion by SuperAtraction** [OlopTech/Olop#5] [Polls]
_**Ce message (Et le "Poll") est mis à jour de temps en temps**_

Voici le futur d'Olop : 
Alpha 2.0 : Nouvelle interface InstallPass (technologie d'installation par code) complète (et début de la personnalisation ?)
Alpha 3.0 : Launcher personnalisable, fonctions Linux, onglets (à confirmer, sinon pour la 3.1)
Alpha 4.0 : Créateur d'applis ?

Pour la Beta 1.0 (pas avant mi-2024, et on est en mi-2023) (v0.1.0-beta) : Launcher avec intégration GPT 3.5 (ou 4 ?), navigateur web avec technologie anti-fake news, correcteur d'erreur Qt c++ avec GPT(?), interface personnalisable à 100 %, paramètres comme Steam big picture (wifi, audio, affichage, etc), un mode jeu (?), onglets.

**Comment by fberanger** [comment]
Bonjour
SuperAtraction, je n'ai pas tout compris, mais j'ai réussi à m'inscrire ! 
_Message modifié par Olop. Raison : Vie privée & sécurité_
